src/components/Home/CareerPart2.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Data from "./JobsData.json";
+import "../styles/careersection.css";
+
+const CareerPart2 = () => {
+    return(
+        <div className="div">
+            <h1 className="header">current open positions</h1>
+            <h3 className="h3">Please send us an email with the application title as the subject with an attached CV in PDF format!</h3>
+            <table className="sm:justify-center ml-[2%]">
+           {
+            
+            Data && Data.map( record => {
+                return(
+                    <div className="sm:justify-center">
+                    <tr key={record.id}>

```

Comment:
The key attribute should be in the first line of the return statement to wrap all the code. Delete it from here and check the above comment